### DISCUSSION TITLE
'Notice' log level not available in profiler

### DISCUSSION THREAD

**Discussion by jdevinemt** [symfony/symfony#45618] [Q&A] [answered]
I'm trying to move my application towards using the monolog bundle for logging instead of it's current logging mechanism. I noticed that the 'Notice' log level is not selectable in the profiler. Is this the desired functionality? When I open the logging tab the notices are displayed when the page is first rendered, but then disappear. It seems they might be being hidden by a script?

<img width="255" alt="Screen Shot 2022-03-02 at 12 17 40 PM" src="https://user-images.githubusercontent.com/65512359/156434077-85a31333-e875-4210-b85e-d09a57a20f86.png">

**Comment by alessandro-podo** [comment, marked as answer]
https://github.com/symfony/symfony/pull/45351
Does this fix your Problem?

**Reply by jdevinemt** [reply]
Yep, thanks! I was thinking it might be something that was resolved in an update, but hadn't updated my packages in a while. I will keep in mind to do this before posting here in the future. 

I went ahead and updated all my Symfony packages and *most* recipes. I am running into an issue trying to update my `doctrine/doctrine-bundle` recipe. Not really sure how I can resolve it..

<img width="407" alt="Screen Shot 2022-03-02 at 1 42 05 PM" src="https://user-images.githubusercontent.com/65512359/156446328-4a12ecf4-f96b-4e0c-aa8d-f98a6ad9ef8e.png">

**Reply by alessandro-podo** [reply]
no problem.
i would recommend you, to create a new Issue for that.
